### ISSUE TITLE
p.u. to absolute values of transformer x

### ISSUE THREAD

**Issue by ulfmueller** [openego/data_processing#146]
The x of the tranformer are p.u. values in osmTGmod. 
Apparently the tranformers x in PyPSA are also in p.u. values  (see [here](https://pypsa.org/doc/components.html#transformer). Therefore the conversion [here](https://github.com/openego/data_processing/blob/dev/dataprocessing/sql_snippets/ego_dp_powerflow_osmtgmod_to_pypsa.sql#L80-L84) seems to be wrong.

Moreover taking into consideration the right base_MVA be aware that osmTGmod uses 100MVA and PyPSA 1MVA (see [here](https://pypsa.org/doc/design.html#internal-use-of-per-unit))

In other words: we probably have to get rid of the existing p.u. conversion but instead levelize the baseMVA difference, right?

@mariusves could you please check this and eventually change it in a branch with a pull request including a review.... ?

**Comment by ulfmueller** [comment]
@mariusves Are my concerns any qualified? If we fastly hot fix this we might be able to consider this in the new version v0.3...

**Comment by mariusves** [comment]
fixed already in 87e1940
